portal/config/config.py

```diff
@@ -85,7 +85,9 @@ class BaseConfig(object):
     CELERY_IMPORTS = ('portal.tasks',)
     DEBUG = False
     DOGPILE_CACHE_BACKEND = 'dogpile.cache.redis'
-    DOGPILE_CACHE_REGIONS = [('hourly', 3600)]
+    DOGPILE_CACHE_REGIONS = [
+        ('assessment_cache_region', 60*60*2),
```

Comment:
Don't you also have to update the [regions in the decorators](https://github.com/uwcirg/true_nth_usa_portal/tree/develop/portal/models/reporting.py#L27)?

Reply:
Whoops, missed that you did change those